src/main/java/ChatResponseRespone.java

```diff
@@ -0,0 +1,29 @@
+import lombok.Data;
```

Comment:
Using wildcard imports like '*' can lead to naming conflicts and make the code less readable. It's recommended to import specific classes.